_includes/page-header.html

```diff
@@ -1,3 +1,7 @@
 <section class="page__header">
-Page Header
+  <h1 class="page__title">{{ page.title | escape }}</h1>
+  {% assign content = content | strip_newlines %}
+  {%- if content != '' -%}
+  <div class="page__desc">{{ content }}</div>
```

Comment:
If the glossary is using this layout, then the page description might be repeated. 
you might do {% if page.name != 'glossary' %} {{ content }} {% endif %} on the pages.html 
or here, do
 {% if page.name != 'glossary' %} <div class="page__desc">{{ content }}</div> {% else %} <div class="page__desc">{{ site.data.language.glossary_description }}</div> {% endif %}

Reply:
Can you push the glossary branch tomorrow so I can look at it? It's unclear to me why the glossary page wouldn't be using the page header component, so there shouldn't be a conflict. The description should be the "content" section of the markdown page (as you pointed out in the other PR).

Reply:
I should probably rephrase my question. It's mostly concerning the glossary. Where should the glossary description go?